### ISSUE TITLE
documentation versioning

### ISSUE THREAD

**Issue by ypylypenko** [pestphp/docs#275]
Hi. Since version 3 of PEST has already been released, I would like to ask if you are planning to introduce versioning for the documentation on the website, and if any help is needed with this? Currently, it's unfortunately difficult to understand which feature is available in which version in the documentation..

thank you

**Comment by owenvoke** [comment]
The documentation always shows the latest versions functionality, so it will show features from 3.x onwards.

I think it would be nice to have versioning, but not sure if the team want the maintenance overhead of 2 documentations. 🤔 A possible feature we could have is to have badges with `v3.0+` (or similar) that is shown next to features. That way we can see when a specific feature was added. 🤷🏻

**Comment by ypylypenko** [comment]
Thank you for your response! I think adding version badges (e.g., v3.0+) is a great idea and would be very helpful for those who haven’t yet transitioned to version 3 (me 🫣) of the framework. It will definitely make it easier to understand which features are available in which version.

If any assistance is needed with implementing, I’d be happy to help.

**Comment by andho** [comment]
Can you have the v2 docs hosted somewhere at least? After all, version 3 just came out, and the library seems to be very strongly tied to laravel because of `nunomaduro/termwind` so it's not possible to update without updating the framework.

**Comment by dmachiavello** [comment]
Yes, another vote for documentation versioning, like the official Laravel Docs. Some of us don't have control of our environments and cannot upgrade to Pest 3 because of requirements, so seeing a v2 would be great!

**Comment by carlcasbolt** [comment]
Would it be viable to tag this documentation prior to the v3 merge here - https://github.com/pestphp/docs/commit/41a78ca7c014b577aafa873ab994bced427b98a8 and then there is a last state of master pre-v3 changes

**Comment by phlyper** [comment]
Hello 
I have just started maintaining a lagacy project implemented with Laravel 8.0 so systematically the pest v1.* version is the most compatible with the project.
I would like to ask you to tell me from which commit in this doc project I can follow in the doc the pest v1.

**Comment by hettiger** [comment]
Also desperately looking for v1 docs :-D
